src/class-wpml-vc-website-builder-translatable-nodes.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * Class WPML_VC_Website_Builder_Translatable_Nodes
+ */
+class WPML_VC_Website_Builder_Translatable_Nodes implements IWPML_Page_Builders_Translatable_Nodes {
+
+	const SETTINGS_FIELD = 'settings';
+
+	/**
+	 * @var string
+	 */
+	private $settings_field;
+
+	/**
+	 * @var string
+	 */
+	private $type;
+
+	/** @var array */
+	private $nodes_to_translate;
+
+	public function __construct() {
+		$this->settings_field = self::SETTINGS_FIELD;
+		$this->type           = 'tag';
```

Comment:
@adrianosferreira why not use 'tag' as const instead of var?

Reply:
@grey-rsi I'm following the pattern we used in Beaver Builder and Elementor integration modules.

Reply:
fine for me then =)